pkg/database/postgres.go

```diff
@@ -0,0 +1,106 @@
+package database
+
+import (
+	"database/sql"
+
+	"github.com/appuio/appuio-cloud-reporting/pkg/db"
+)
+
+type productDBaaS struct {
+	Plan   string
+	Target string
+	Amount float64
+}
+
+var (
+	// Hobbyist2 plan
+	Hobbyist2 = productDBaaS{Plan: "hobbyist-2", Target: "1411", Amount: 0.05811000}
```

Comment:
[pricing](https://products.docs.vshn.ch/products/appcat/exoscale_dbaas.html#_pricing) mentions a different price for hobbyist 2 (I only checked this one, didn't check other ones). 

Which one is correct? Or are both correct since there's some margin applied later on?

Reply:
good question, and I'm not sure about it, maybe @ZuGao knows?

Reply:
ah that's interesting. I think I got the prices from one of my colleagues on Friday which might be wrong as I'm checking now. @Kidswiss can you please check and let us know here what prices are correct?

Reply:
FYI: the prices we use come from the [Portal Exoscale API](https://portal.exoscale.com/api/pricing/dbaas-pg)
What I saw is that we didn't add plan `hobbyist-1`, can you double checked that it exists in Exoscale? I don't have access to Exoscale, sorry.

Reply:
To ensure the correct prices is in another ticket: https://ticket.vshn.net/browse/APPCAT-264

For this PR it's okay to have dummy values, but if you want you can already get the prices from the CSV in APPCAT-264.

Reply:
perfect, thank you (that's the ones we used). I'll add `hobbyist-1` that was missing 👍

Reply:
@susana-garcia aah, I think you used the wrong column then :)

According to APPCAT-264, the relevant prices are in the "Final Price / hour" column ;)

Reply:
oh, I see. Sorry, I missed that part on the ticket. I'll update them 😄